global_helpers/panther_oss_helpers.py

```diff
@@ -163,265 +162,88 @@ def resource_lookup(resource_id: str) -> Dict[str, Any]:
     return response["Item"]["attributes"]
 
 
-# Helper functions for accessing Dynamo key-value store.
-#
-# Keys can be any string specified by rules and policies,
-# values are integer counters and/or string sets.
-#
-# Use kv_table() if you want to interact with the table directly.
-_KV_TABLE = None
-_COUNT_COL = "intCount"
-_STRING_SET_COL = "stringSet"
-_DICT_COL = "dictionary"
-_TTL_COL = "expiresAt"
-
-
-def kv_table() -> boto3.resource:
-    """Lazily build key-value table resource"""
-    # pylint: disable=global-statement
-    global _KV_TABLE
-    if not _KV_TABLE:
-        # pylint: disable=no-member
-        _KV_TABLE = boto3.resource(
-            "dynamodb",
-            endpoint_url="https://dynamodb" + FIPS_SUFFIX if FIPS_ENABLED else None,
-        ).Table("panther-kv-store")
-    return _KV_TABLE
-
-
 def ttl_expired(response: dict) -> bool:
-    """Checks whether a response from the panther-kv table has passed it's TTL date"""
-    # This can be used when the TTL timing is very exacting and DDB's cleanup is too slow
-    expiration = response.get("Item", {}).get(_TTL_COL, 0)
-    return expiration and float(expiration) <= (datetime.now()).timestamp()
+    """Global `ttl_expired` is DEPRECATED.
+    Instead, use `panther_detection_helpers.caching.ttl_expired`."""
```

Comment:
Should this be something like:
```
from panther_detection_helpers import caching
caching.ttl_expired
```
to match what you're doing here?

Reply:
Hmm that is interesting. I was just pointing to the thing they should be using. But I could see how import directions could also be helpful